containers/ddev-ssh-agent/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM bitnami/minideb:buster
+FROM debian:buster-slim
```

Comment:
FYI I did a simple test of difference of size between using a base image of debian:buster-slim and debian:buster the other day when debugging one of the build problems we had (wondering if slim was a problem). It turned out that even though we add enormously to the image, there's still a 50MB or so benefit (as I remember) of using buster-slim.